Q21/StudentProbability.rb

```diff
@@ -0,0 +1,48 @@
+def first_game
+	check = false
+	stack = []
+	counter = 0
+	roll = 0
+	until check == true
+	roll = rand(1..6)
+		if roll == 5 && stack.empty?
+			stack.push(roll)
+			# p stack
+		elsif roll == 6 && stack.pop == 5
+			# p "Happened one"
+			stack.push(roll)
+			check = true
+		else
+			stack.clear
+		end
+		counter += 1
+	end
+	# 	p "second #{stack}"
+	p "first game counter: #{(counter)*1/36.to_f}"
```

Comment:
Why are you multiplying by 1/36 and then to_f do?

Reply:
If I don't have the .to_f to parse and show it as a fraction it won't display the number a a fraction only as a whole number